diff --git a/dist/index.js b/dist/index.js
index 831428320436f81e6d1c4f2698d55335fa72f900..418b677ba1dd68ae107a39bc7f646d1316236463 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,7 +1,7 @@
 // Export Kernels
 export { ObjectKernel } from '@objectstack/core';
 // Export Plugins
-export { DriverPlugin } from './driver-plugin';
-export { AppPlugin } from './app-plugin';
+export { DriverPlugin } from './driver-plugin.js';
+export { AppPlugin } from './app-plugin.js';
 // Export Types
 export * from '@objectstack/core';
